client/src/contexts/ItemsContext.jsx

```diff
@@ -138,12 +140,27 @@ export function useItems() {
 
 // Provides the store to your application
 export function ItemsProvider({ children }) {
+  const [items, setItems] = useState('')
+
+  useEffect(() => {
+    fetch('http://localhost:4001/items')
```

Comment:
Since we'll be fetching from the same backend URL each time (and this backend URL may change) it might be nice to have a `BACKEND_URL` variable in a config file. That way we can call fetch like `fetch(`${BACKEND_URL}/items` instead.

Reply:
Hey, Abdul. I'm not really sure where to make that variable. Would love to know which config file I should put the code in. 

If I remember correctly, when you set up typescript, you used an env file. Would that be the same with this one?